### PR TITLE
[CCV] CUR 2.0 - update public docs following CUR changes

### PR DESCRIPTION
**Pull request by crousseaux**
### What does this PR do? What is the motivation?
AWS released CUR 2.0, we need to update all setup/documentation of “CUR” to the “Legacy CUR”

More details here: [CUR 2.0](https://docs.google.com/document/d/18I_jEIR65bK3H26AojMsA9crqDW-kID6GFyFpOe3LAQ/edit#heading=h.vw24el6pa3ln)

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/aws.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ further_reading:
 
 ## Overview
 
-To use AWS Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a Cost and Usage report.
+To use Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a legacy Cost and Usage report.
```
Comment (cnharris10) [inline content/en/cloud_cost_management/aws.md:22]:
```suggestion
To use Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a legacy Cost and Usage report through AWS Billing.
```

Comment (cnharris10) [reply content/en/cloud_cost_management/aws.md:22]: Targeting this: 
image

and not this:
image

**Review by cnharris10 — CHANGES_REQUESTED** [review]

File: content/en/cloud_cost_management/aws.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ further_reading:
...
 
 ## Setup
 
-### Prerequisite: generate a Cost and Usage Report
+### Prerequisite: generate a Legacy Cost and Usage Report
```
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:26]:
```suggestion
### Prerequisite: generate a legacy Cost and Usage Report
```

File: content/en/cloud_cost_management/aws.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ Navigate to [Setup & Configuration][7] and select an AWS account from the dropdo
 
 This ensures complete cost accuracy by allowing periodic cost calculations against the AWS Cost Explorer.
 
-### Locate the Cost and Usage Report
+### Locate the Legacy Cost and Usage Report
```
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:53]:
```suggestion
### Locate the legacy Cost and Usage Report
```

File: content/en/cloud_cost_management/aws.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ To enable Datadog to locate the Cost and Usage Report, complete the fields with
 
 **Note**: Datadog only supports CURs generated by AWS. Do not modify or move the files generated by AWS, or attempt to provide access to files generated by a 3rd party.
 
-### Configure access to the Cost and Usage Report
+### Configure access to the Legacy Cost and Usage Report
```
Comment (urseberry) [inline content/en/cloud_cost_management/aws.md:66]:
```suggestion
### Configure access to the legacy Cost and Usage Report
```

**Review by urseberry — CHANGES_REQUESTED** [review]

File: content/en/cloud_cost_management/aws.md [hunk 5]
```diff
@@ -19,13 +19,18 @@ further_reading:
...
+1. Have an AWS account with billing access
+2. Have the AWS integration installed in Datadog
+3. Follow the steps below to create a Cost and Usage report
 
 ## Setup
 
 ### Prerequisite: generate a Cost and Usage Report
 
-Follow AWS instructions for [Creating Cost and Usage Reports][1], and select the following content options for use with Datadog Cloud Cost Management:
+[Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. You cannot create a report with data exports.
```
Comment (cnharris10) [inline content/en/cloud_cost_management/aws.md:31]:
```suggestion
[Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. At this time, there is no support for creating a Cost and Usage Report data exports.
```

**Review by cnharris10 — APPROVED** [review]

File: content/en/cloud_cost_management/aws.md [hunk 6]
```diff
@@ -19,13 +19,18 @@ further_reading:
...
+1. Have an AWS account with billing access
+2. Have the AWS integration installed in Datadog
+3. Follow the steps below to create a Cost and Usage report
 
 ## Setup
 
 ### Prerequisite: generate a Cost and Usage Report
 
-Follow AWS instructions for [Creating Cost and Usage Reports][1], and select the following content options for use with Datadog Cloud Cost Management:
+[Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. At this time, there is no support for creating a Cost and Usage Report data exports.
```
Comment (cnharris10) [inline content/en/cloud_cost_management/aws.md:31]:
```suggestion
[Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. At this time, there is no support for creating Cost and Usage Report data exports.
```

File: content/en/cloud_cost_management/aws.md [hunk 7]
```diff
@@ -19,13 +19,18 @@ further_reading:
 
 ## Overview
 
-To use AWS Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a Cost and Usage report.
+To set up Cloud Cost Management in Datadog, you should:
+1. Have an AWS account with billing access
+2. Have the AWS integration installed in Datadog
+3. Follow the steps below to create a Cost and Usage report
```
Comment (alai97) [inline content/en/cloud_cost_management/aws.md:25]:
```suggestion
1. Have an AWS account with billing access.
2. Have the AWS integration installed in Datadog.
3. Follow the steps below to create a Cost and Usage report.
```

File: content/en/cloud_cost_management/aws.md [hunk 8]
```diff
@@ -19,13 +19,18 @@ further_reading:
...
 
 ## Setup
 
 ### Prerequisite: generate a Cost and Usage Report
 
-Follow AWS instructions for [Creating Cost and Usage Reports][1], and select the following content options for use with Datadog Cloud Cost Management:
+[Create a Cost and Usage Report][1] in AWS under the **Legacy Pages** section. At this time, there is no support for creating Cost and Usage Report data exports.
```
[the remark this answers is not in the document]
Comment (crousseaux) [reply content/en/cloud_cost_management/aws.md:31]: AWS released last week a new way of generating billing exports called "data exports". The previous way that we support has been moved under the "Legacy Pages" section.
We want to make it clear that they should continue to use the legacy way. We plan to provide support for data exports in the near future.

**Review by urseberry — APPROVED** [review]

**Pull request merged**